### PR TITLE
add DIGESTS flag for make csv

### PR DESCRIPTION
**Pull request by tchughesiv**
Signed-off-by: tchughesiv <tchughesiv@gmail.com>

### CODE REVIEW

File: deploy/catalog_resources/community/1.4/kiecloud-operator.1.4.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -167,17 +167,17 @@ spec:
                 - name: PAM_SMARTROUTER_IMAGE_7.7.0
                   value: registry.redhat.io/rhpam-7/rhpam-smartrouter-rhel8:7.7.0
                 - name: OSE_CLI_IMAGE_7.7.0
-                  value: registry.redhat.io/openshift3/ose-cli@sha256:7e356132f56fc4e3cf691ba5b0b5f063dc974c4cfe2f198c8c1b5f68cb9ccbdd
+                  value: registry.redhat.io/openshift3/ose-cli:v3.11
```
Comment (bmozaffa) [inline deploy/catalog_resources/community/1.4/kiecloud-operator.1.4.0.clusterserviceversion.yaml]: Are you building with the flag off?

Comment (tchughesiv) [reply deploy/catalog_resources/community/1.4/kiecloud-operator.1.4.0.clusterserviceversion.yaml]: yes ... let me do so with flag on instead.

Comment (tchughesiv) [reply deploy/catalog_resources/community/1.4/kiecloud-operator.1.4.0.clusterserviceversion.yaml]: done

**Pull request merged**